cime_config/acme/machines/config_compilers.xml

```diff
@@ -790,6 +790,9 @@ for mct, etc.
 </compiler>
 
 <compiler OS="Linux" COMPILER="intel">
+  <CPPDEFS>
```

Comment:
Rather than hard coding it this way we should use the OS and COMPILER variables.